user/languages/erlang.md

```diff
@@ -6,10 +6,38 @@ layout: en
 
 ### What This Guide Covers
 
-This guide covers build environment and configuration topics specific to Erlang projects. Please make sure to read our [Getting Started](/user/getting-started/) and [general build configuration](/user/customizing-the-build/) guides first.
+> Language versions and other build-environment specific
+> information are in our reference pages:
```

Comment:
could we update this to the existing snippet? 

Adding the line `{{ site.data.snippets.trusty_note_no_osx }}`

will add the following note: 

![image](https://user-images.githubusercontent.com/3729517/30112050-3e803f8e-9310-11e7-97ff-64e3263b5621.png)
 :tada